MoviesMVVM/MoviesMVVM/DetailMoviesViewController/ActorCollectionViewCell.swift

```diff
@@ -53,13 +57,15 @@ final class ActorCollectionViewCell: UICollectionViewCell {
         setConstraints()
     }
 
-    // MARK: Public Methods
+    // MARK: - Public Methods
 
-    func setCellWithValues(_ actor: ActorInfo) {
-        setUI(actorImage: actor.actorImage, actorName: actor.name)
+    func setCellWithValues(actor: ActorInfo, imageURL: String, viewModel: DetailsMovieViewModelProtocol) {
+        actorNameLabel.text = actor.name
+        let urlString = "\(NetworkAPI.imageURL)\(imageURL)"
```

Comment:
```suggestion
        let urlString = "\\(imageURL)"
```
ячейка не знает ни о каких API